nemo/collections/asr/parts/segment.py

```diff
@@ -1,5 +1,18 @@
 # Taken straight from Patter https://github.com/ryanleary/patter
```

Comment:
This code seems to be MIT:
https://github.com/ryanleary/patter/blob/master/LICENSE

Should we attach the MIT header too?

Reply:
That is a great question. I have no idea.

Reply:
actually, yes, we do. @blisc  feel free to add it in this PR or I can add it later myself

Reply:
With @ekmb we were following this pattern:
 - NVIDIA license header on top
 - other license headers below

https://github.com/NVIDIA/NeMo/blob/master/nemo/collections/nlp/nm/trainables/dialogue_state_tracking/trade_generator_nm.py

Plus we came up with this formula:
```
# This file contains code artifacts adapted from the original implementation:
# https://github.com/thu-coai/ConvLab-2/blob/master/convlab2/nlg/template/multiwoz/nlg.py
```
https://github.com/NVIDIA/NeMo/blob/master/nemo/collections/nlp/nm/non_trainables/dialogue_state_tracking/template_nlg_multiwoz.py#L17

IMO this is super fair (we are giving credits to the original authors) and super-secure (nobody can say that we violated anything).

Reply:
Done in #989